ERROR [main] portico.lrc: FAILURE Exception encountered while processing incoming message: discovered multiple object instances with handle 2097155
hla.rti.FederateInternalError: discovered multiple object instances with handle 2097155 serial:0
	at gov.nist.hla.ii.FederateAmbassador.discoverObjectInstance(FederateAmbassador.java:137)
	at org.portico.impl.hla13.handlers.DiscoverObjectCallbackHandler.process(DiscoverObjectCallbackHandler.java:69)
	at org.portico.utils.messaging.MessageChain.process(MessageChain.java:78)
	at org.portico.utils.messaging.MessageSink.process(MessageSink.java:187)
	at org.portico.lrc.LRC.tickProcess(LRC.java:688)
	at org.portico.lrc.LRC.tickUntilEmpty(LRC.java:489)
	at org.portico.lrc.LRC.tick(LRC.java:449)
	at org.portico.impl.hla13.Impl13Helper.tick(Impl13Helper.java:130)
	at org.portico.impl.hla13.Rti13Ambassador.tick(Rti13Ambassador.java:5781)
	at gov.nist.hla.ii.InjectionFederate.tick(InjectionFederate.java:337)
	at gov.nist.hla.ii.InjectionFederate.synchronize(InjectionFederate.java:695)
	at gov.nist.hla.ii.InjectionFederate.run(InjectionFederate.java:209)
	at gov.nist.hla.ii.GatewayMain.main(GatewayMain.java:25)
ERROR [main] portico.lrc: FAILURE Exception encountered while processing incoming message: discovered multiple object instances with handle 2097153
hla.rti.FederateInternalError: discovered multiple object instances with handle 2097153 serial:0
	at gov.nist.hla.ii.FederateAmbassador.discoverObjectInstance(FederateAmbassador.java:137)
	at org.portico.impl.hla13.handlers.DiscoverObjectCallbackHandler.process(DiscoverObjectCallbackHandler.java:69)
	at org.portico.utils.messaging.MessageChain.process(MessageChain.java:78)
	at org.portico.utils.messaging.MessageSink.process(MessageSink.java:187)
	at org.portico.lrc.LRC.tickProcess(LRC.java:688)
	at org.portico.lrc.LRC.tickUntilEmpty(LRC.java:489)
	at org.portico.lrc.LRC.tick(LRC.java:449)
	at org.portico.impl.hla13.Impl13Helper.tick(Impl13Helper.java:130)
	at org.portico.impl.hla13.Rti13Ambassador.tick(Rti13Ambassador.java:5781)
	at gov.nist.hla.ii.InjectionFederate.tick(InjectionFederate.java:337)
	at gov.nist.hla.ii.InjectionFederate.synchronize(InjectionFederate.java:695)
	at gov.nist.hla.ii.InjectionFederate.run(InjectionFederate.java:209)
	at gov.nist.hla.ii.GatewayMain.main(GatewayMain.java:25)
ERROR [main] portico.lrc: FAILURE Exception encountered while processing incoming message: discovered multiple object instances with handle 2097155
hla.rti.FederateInternalError: discovered multiple object instances with handle 2097155 serial:0
	at gov.nist.hla.ii.FederateAmbassador.discoverObjectInstance(FederateAmbassador.java:137)
	at org.portico.impl.hla13.handlers.DiscoverObjectCallbackHandler.process(DiscoverObjectCallbackHandler.java:69)
	at org.portico.utils.messaging.MessageChain.process(MessageChain.java:78)
	at org.portico.utils.messaging.MessageSink.process(MessageSink.java:187)
	at org.portico.lrc.LRC.tickProcess(LRC.java:688)
	at org.portico.lrc.LRC.tickUntilEmpty(LRC.java:489)
	at org.portico.lrc.LRC.tick(LRC.java:449)
	at org.portico.impl.hla13.Impl13Helper.tick(Impl13Helper.java:130)
	at org.portico.impl.hla13.Rti13Ambassador.tick(Rti13Ambassador.java:5781)
	at gov.nist.hla.ii.InjectionFederate.tick(InjectionFederate.java:337)
	at gov.nist.hla.ii.InjectionFederate.synchronize(InjectionFederate.java:695)
	at gov.nist.hla.ii.InjectionFederate.run(InjectionFederate.java:209)
	at gov.nist.hla.ii.GatewayMain.main(GatewayMain.java:25)
ERROR [main] portico.lrc: FAILURE Exception encountered while processing incoming message: discovered multiple object instances with handle 2097153
hla.rti.FederateInternalError: discovered multiple object instances with handle 2097153 serial:0
	at gov.nist.hla.ii.FederateAmbassador.discoverObjectInstance(FederateAmbassador.java:137)
	at org.portico.impl.hla13.handlers.DiscoverObjectCallbackHandler.process(DiscoverObjectCallbackHandler.java:69)
	at org.portico.utils.messaging.MessageChain.process(MessageChain.java:78)
	at org.portico.utils.messaging.MessageSink.process(MessageSink.java:187)
	at org.portico.lrc.LRC.tickProcess(LRC.java:688)
	at org.portico.lrc.LRC.tickUntilEmpty(LRC.java:489)
	at org.portico.lrc.LRC.tick(LRC.java:449)
	at org.portico.impl.hla13.Impl13Helper.tick(Impl13Helper.java:130)
	at org.portico.impl.hla13.Rti13Ambassador.tick(Rti13Ambassador.java:5781)
	at gov.nist.hla.ii.InjectionFederate.tick(InjectionFederate.java:337)
	at gov.nist.hla.ii.InjectionFederate.synchronize(InjectionFederate.java:695)
	at gov.nist.hla.ii.InjectionFederate.run(InjectionFederate.java:209)
	at gov.nist.hla.ii.GatewayMain.main(GatewayMain.java:25)
ERROR [main] portico.lrc: FAILURE Exception encountered while processing incoming message: discovered multiple object instances with handle 2097155
hla.rti.FederateInternalError: discovered multiple object instances with handle 2097155 serial:0
	at gov.nist.hla.ii.FederateAmbassador.discoverObjectInstance(FederateAmbassador.java:137)
	at org.portico.impl.hla13.handlers.DiscoverObjectCallbackHandler.process(DiscoverObjectCallbackHandler.java:69)
	at org.portico.utils.messaging.MessageChain.process(MessageChain.java:78)
	at org.portico.utils.messaging.MessageSink.process(MessageSink.java:187)
	at org.portico.lrc.LRC.tickProcess(LRC.java:688)
	at org.portico.lrc.LRC.tickUntilEmpty(LRC.java:489)
	at org.portico.lrc.LRC.tick(LRC.java:449)
	at org.portico.impl.hla13.Impl13Helper.tick(Impl13Helper.java:130)
	at org.portico.impl.hla13.Rti13Ambassador.tick(Rti13Ambassador.java:5781)
	at gov.nist.hla.ii.InjectionFederate.tick(InjectionFederate.java:337)
	at gov.nist.hla.ii.InjectionFederate.synchronize(InjectionFederate.java:695)
	at gov.nist.hla.ii.InjectionFederate.run(InjectionFederate.java:209)
	at gov.nist.hla.ii.GatewayMain.main(GatewayMain.java:25)
